roles/openstack/nova/tasks/setup.yml

```diff
@@ -0,0 +1,43 @@
+---
+# This file is used to create OpenStack nova resources
+
+###todo(abregman): Enable 'create_flavors' when ansible 2.1 is out
+#- name: Create flavors
+#  os_nova_flavor:
+#      name: "{{ provisioner.prefix | default(random.uuid) }}-{{ item.value.nane | default(flavor_name) }}"
+#      ram: "{{ item.value.ram | default(omit) }}"
+#      disk: "{{ item.value.disk | default(omit) }}"
+#      vcpus: "{{ item.value.vcpus | default(omit) }}"
+#      state: present
+#  register: flavors
+#  with_dict: "{{ provisioner.flavors }}"
+#  when: provisioner.flavors is defined  
+
+- name: Create instances
+  os_server:
+      cloud: "{{ provisioner.cloud }}"
```

Comment:
Please use `default(omit)` for cloud value to allow users to source "keystonerc" files. This is a great functionality in shade in the UX front.